src/data/faq_de.json

```diff
@@ -448,6 +448,18 @@
                             "<ol><li>Öffnen Sie die Geräte-Einstellungen</li><li>Wählen Sie 'Sicherheit & Standort' -> 'Standort' -> 'Berechtigungen auf App-Ebene'</li></ol>",
                             "Hier können Sie den installierten Apps die Berechtigung für den Standortzugriff entziehen, wenn Sie denken, dass diese nicht benötigt wird. Die Corona-Warn-App greift nicht auf Ihren Standort zu, deshalb ist sie nicht aufgeführt."
                         ]
+                    },
+                    {
+                        "title": "[Google/Android] Auf meinem Hauptbildschirm werde ich gewarnt, dass mein Gerät nicht oder nicht vollständig kompatibel ist. Was bedeutet das?",
+                        "anchor": "incompatibility_warning",
+                        "active": true,
+                        "textblock": [
+                            "Wenn Sie diese Warnmeldung sehen, ist Ihr Gerät nicht in der Lage, Zufalls-IDs von anderen Geräten zu empfangen. Dies wird durch eine Einschränkung in der Kombination aus verwendeter Hard- und Software verursacht. Dadurch kann Bluetooth-Low-Energy im sogenannten „peripheral mode“ nicht von der Corona-Warn-App verwendet werden.",

```

Comment:
statt: 

Wenn Sie diese Warnmeldung sehen, ist Ihr Gerät nicht in der Lage, Zufalls-IDs von anderen Geräten zu empfangen.

bitte:

Wenn Sie diese Warnmeldung sehen, ist Ihr Gerät nicht in der Lage, Zufalls-IDs an andere Geräte zu senden.

Damit passt die Übersetzung, und sie beschreibt auch das tatsächliche Problem. Das Phone sendet keine Beacons, andere Phones können daher nichts empfangen. Ob das eigene Phone Beacons empfangen kann, und ob die CWA diese dann auch sieht, ist schwer zu testen ohne Feedback durch CWA.

Reply:
Good catch. I'm sure Fynn only mixed the words despite having the true direction in mind.
Maybe it could also be helpful to more emphasize the both conditions (partially incompatible, fully incompatible), by changing the first sentence a little:

Wenn Sie eine Warnmeldung sehen, dass Ihr Gerät teilweise nicht kompatibel ist, ist es nicht in der Lage, Zufalls-IDs an andere Geräte zu senden.

Reply:
Right, I cut off the wrong part of the sentence from our [German CCTG FAQ](https://codeberg.org/corona-contact-tracing-germany/cwa-android/src/branch/main/FAQ_de.md#auf-meinem-hauptbildschirm-werde-ich-gewarnt-dass-mein-ger%C3%A4t-nicht-vollst%C3%A4ndig-kompatibel-ist-was-bedeutet-das). Thanks for noticing @pinkit-cwa.

@vaubaehn The message applies to both partial as well as full incompatibility.

Reply:
> @vaubaehn The message applies to both partial as well as full incompatibility.

You're completely right. Sometimes I need the "Holzhammer", but after reading again, I think it's good like it is.